skrf/network.py

```diff
@@ -6564,6 +6766,12 @@ def impedance_mismatch(z1: NumberLike, z2: NumberLike) -> npy.ndarray:
             complex impedance of port 1
     z2 : number or array-like
             complex impedance of port 2
+    s_def : str
+        Scattering parameter definition:
+        'pseudo' for pseudo-waves definition.
+        'traveling' corresponds to the initial implementation.
+        'power' is not supported.
```

Comment:
```suggestion
        'power' is not supported.
```
well, it could in case of real-valued reference impedance.

Did you use a reference for the equations, to add it in the docstring?

Reply:
It's not directly in this form but the T-matrix for pseudo impedance mismatch is from: R. B. Marks et D. F. Williams, "A general waveguide circuit theory". I will add it to docstring.